src/pages/WalletsImport/walletsImport.m.scss

```diff
@@ -1,7 +1,10 @@
 @import 'styles/mixins';
 
 .core {
+  @include flex-row-center;
+
   width: 100%;
+  height: 100%;
```

Comment:
Looks ok on narrow screens?

Reply:
Is there any place with etalon sketches for narrow screens? Which we can compare with our results?

Reply:
No, we just check it for sanity right now. If it doesn't look bad or broken, it's fine